### PR TITLE
Added healthchecks and update the readme

### PR DESCRIPTION
**Pull request by AshenW0lf**
See the readme for details of the changes

### CODE REVIEW

File: src/Marvin.Web/HealthChecks/ExampleHealthCheck.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Marvin.Web.HealthChecks
+{
+    public class ExampleHealthCheck : IHealthCheck
+    {
+        public Task<HealthCheckResult> CheckHealthAsync(
+            HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            var isHealthy = true;
```
Comment (tonyjoanes) [inline src/Marvin.Web/HealthChecks/ExampleHealthCheck.cs:10]: We could `Match` on this:

```csharp
return isHealthy.Match(
   t => Task.FromResult(HealthCheckResult.Healthy("A healthy result.")),
   f => Task.FromResult(new HealthCheckResult(context.Registration.FailureStatus, "An unhealthy result."))
);
```

> this is the sort of stuff we do with functional programming style, just showing this as an example, it becomes more powerful when done over types

File: src/Marvin.Web/Program.cs [hunk 2]
```diff
@@ -40,6 +43,15 @@
 
 builder.Services.AddRazorPages();
 
+builder.Services
+    .AddHealthChecksUI()
+    .AddInMemoryStorage();
+/*    .AddPostgreSqlStorage(connectionString);*/
+
+builder.Services.AddHealthChecks()
+    .AddNpgSql(connectionString, name: "Database")
```
Comment (tonyjoanes) [inline src/Marvin.Web/Program.cs:52]: I guess we will get an error now if we did use the mssql connection but ok if we just go with PostgreSQL

**Review by tonyjoanes — APPROVED** [review]

File: src/Marvin.Web/HealthChecks/ExampleHealthCheck.cs (hunk 1 above)
Comment (AshenW0lf) [reply src/Marvin.Web/HealthChecks/ExampleHealthCheck.cs:11]: I have now updated this, in line with the comment.

File: src/Marvin.Web/Program.cs (hunk 2 above)
Comment (AshenW0lf) [reply src/Marvin.Web/Program.cs]: Good point and I missed that,  I've now added a database bootstrapper so both SQL Server and Postgres should work

**Review by tonyjoanes — APPROVED** [review]
Nice 😍

**Pull request merged**